src/table.js

```diff
@@ -314,6 +315,74 @@ export const addColumnAt = columnIndex => tr => {
   return tr;
 };
 
+// :: (originRowIndex: number, targetRowIndex: targetColumnIndex) → (tr: Transaction) → Transaction
+// Returns a new transaction that moves the origin column to the target position
+//
+// ```javascript
+// dispatch(
+//   moveRow(x, y)(state.tr)
+// );
+// ```
+export const moveRow = (originRowIndex, targetRowIndex) => tr => {
+  const table = findTable(tr.selection);
+  if (!table) {
+    return tr;
+  }
+
+  const rangeOriginRow = getSelectionRangeInRow(originRowIndex)(tr);
+  const { indexes: indexesOriginRow } = rangeOriginRow;
```

Comment:
1. You could do that in one line
```
const { indexes: indexesOriginRow }  = getSelectionRangeInRow(originRowIndex)(tr);
```
2. could you add a comment about what `indexesOriginRow` is used for?

Reply:
1. good catch
2. Same as before, I didn't see any comment to explain code on this repo, do you think I should add more unit tests explaining it?